### ISSUE TITLE
Nombre por defecto de espectros que sea real

### ISSUE THREAD

**Issue by facundoq** [midusi/PlateUNLP#67]
Está implementado como que si el nombre es "" entonces se muestra nuevo#indice. Cambiar eso por directamente ponerle el nombre Nuevo#1 y listo.

Entiendo que lo usan para validar. En lugar de eso, en la validación, fijarse si el nombre es de formato "Nuevo#{num}" y si es así declarar ese campo como inválido.

**Comment by joaquinmir** [comment]
El tema es que hacerlo asi genera unos problemas con la actualizacion de los componentes, es mas por eso que por la validacion.

**Comment by facundoq** [comment]
¿Por qué sería? Si el valor inicial al crear el objeto es distinto de ""
¿cuál sería la diferencia con los campos pre-llenados como SCANNER?

**Comment by joaquinmir** [comment]
No es un problema con la logica de la app, digo internamente no hay problema con iniciar los espectros con Nuevo#x, el tema es que con los componentes se complica la cosa. Por ejemplo si tenes Nuevo#1, Nuevo#2 y Nuevo#3. Si borro Nuevo#2 con esa logica me quedaria Nuevo#1, Nuevo#3. Y para hacer que quede bien necesitaria mas logica que lo haria menos eficiente. Ademas quedaria en el input del form Nuevo#1 por ejemplo y no se si es lo mejor, de ultima lo vemos mañana.

**Comment by facundoq** [comment]
No, no habría drama con eso, de hecho mejor porque se entiende que borraste
el 2 y quedaron el 1 y el 3, total todo eso debería cambiarse a un nombre
más específico!

**Comment by joaquinmir** [comment]
Ya hice el cambio, despues checkeenlo porque como comentaba se pueden dar casos en los que se repitan los nombres.
